src/Stan/Config.hs

```diff
@@ -119,12 +122,14 @@ defaultConfig :: PartialConfig
 defaultConfig = ConfigP
     { configChecks  = withTag "Default" $ pure []
     , configRemoved = withTag "Default" $ pure []
+    , configObservations = withTag "Default" $ pure []
     }
 
 finaliseConfig :: PartialConfig -> Trial Text Config
 finaliseConfig config = do
     configChecks  <- #configChecks config
     configRemoved <- #configRemoved config
+    configObservations <- #configObservations config
     pure ConfigP {..}
```

Comment:
I like to see how the code changes after adding new fields :slightly_smiling_face: